### DISCUSSION TITLE
geo::Point support

### DISCUSSION THREAD

**Discussion by kushwahashiv** [diesel-rs/diesel#4121] [Ideas]
### Versions

- **Rust:**
- **Diesel:2.2.1
- **Database:**
- **Operating System**

### Feature Flags

- **diesel:**

## Problem Description
is geo::Point is supported in diesel schema?
if not supported. what are the best options available to geo-coordinates? 

### What are you trying to accomplish?
Trying to use longitude/latitude 

### What is the expected output?

geo::Point should be supported if possible 

## Checklist

- [ ] I have already looked over the [issue tracker](https://github.com/diesel-rs/diesel/issues) and the [discussion forum](https://github.com/diesel-rs/diesel/discussions) for similar possible closed issues.
 
- [ ] This issue can be reproduced on Rust's stable channel. (Your issue will be
  closed if this is not the case)
- [ ] This issue can be reproduced without requiring a third party crate

**Comment by weiznich** [comment]
Generally speaking: Our issue tracker is not a support forum, but a tool to track bugs for contributors. If you want to ask a question, please use the support forum (or other support channels) instead. This time I converted your issue to a discussion. I will treat future questions filled as issues as bug reports, which would in this case mean that I just close them without any comments (as it's clearly not a bug in an existing feature).

That all written: Diesel does not provide built-in support for postgresql extensions. It's designed to be extended by third party crates for this use-case. We demonstrate how such a crate would look like by providing [diesel-full-text-search](https://github.com/diesel-rs/diesel_full_text_search/). For postgis there exist various third party crates: https://crates.io/search?q=postgis. I cannot comment on which one works best or might cover your usecase.

**Comment by fernandomarca** [comment]
@kushwahashiv I did my own implementation and published the crate on crates.io. (Only Postgres support.)
https://crates.io/crates/diesel-point
